src/Modes/imu_downlink_type.enum

```diff
@@ -0,0 +1,11 @@
+#ifndef imu_downlink_type_enum_
+#define imu_downlink_type_enum_
+
+enum class imu_downlink_type : unsigned char
+{
+    GAUSS_8,
+    GAUSS_12,
+    GAUSS_16
+};
+
+#endif
```

Comment:
If you could also create modes for the magnetorquer and accelerometer settings that would also be great. Look at IMU monitor for the names.